src/SFA.DAS.EAS.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using SFA.DAS.EAS.Account.Api.Attributes;
+using SFA.DAS.EAS.Application.Services.EmployerAccountsApi;
+
+namespace SFA.DAS.EAS.Account.Api.Controllers
+{
+    [ApiAuthorize(Roles = "ReadUserAccounts")]
+    [RoutePrefix("api/user")]
+    public class UserController : ApiController
+    {
+        private readonly IEmployerAccountsApiService _apiService;
+
+        public UserController(IEmployerAccountsApiService apiService)
+        {
+            _apiService = apiService;
+        }
+
+        [Route("")]
```

Comment:
I know this is legit, but it does look a bit hacky, can we not get away with just `[HttpGet]` and then allow it to inherit the route from the controller attribute. I know i'm being picky.....